content/en/platform/corda/5.0/developing-applications/api/api-crypto.md

```diff
@@ -8,15 +8,21 @@ menu:
     parent: corda5-api
     weight: 5000
 section_menu: corda5
-draft: true
 ---
 # net.corda.v5.crypto
 
-The `corda-crypto` module 
+The `crypto` package provides services and types for performing cryptographic operations. The following services are available:
+
+* `CompositeKey` is a tree data structure that enables the representation of composite public keys, which are used to represent the signing requirements for multi-signature scenarios.
+* `DigestAlgorithmName` is used in the Corda hashing API.
+* `DigitalSignature` is used to identify the owner of the signing key used to create the signature.
+* `SecureHash` is a cryptographically secure hash value, computed by a specified digest algorithm.
+* The `SignatureSpec` is a digital signature scheme.
 
 ## Implementing Signature Schemes
```

Comment:
This should rather be "Implemented" or maybe "Offered" I think.

Also maybe not for this PR but I am not sure this should go here? This sounds to me more like implementation and this page is about API. Maybe we could consider moving it elsewhere? The only thing we need at this point is mention `SignatureSpec` which we do further above.